app-backend/common/src/test/scala/com/implicits/Generators.scala

```diff
@@ -1224,5 +1255,15 @@ object Generators extends ArbitraryInstances {
       Arbitrary {
         taskPropertiesCreateGen
       }
+
+    implicit def arbStacExportCreate: Arbitrary[StacExport.Create] =
+      Arbitrary {
+        stacExportCreateGen
+      }
+
+    implicit def arbStacExportQueryParameters: Arbitrary[StacExportQueryParameters] =
```

Comment:
After running ./scripts/test, sbt formatted a bunch of lines in this file and in StacExportDaoSpec.scala

Can you commit those changes?